examples/openshift/clamav/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openshift/origin:latest
+
+# Reset the entrypoint
+ENTRYPOINT ["/bin/sh", "-c"]
+
+RUN yum install -y clamav-server clamav-scanner clamav-data --disablerepo=origin-local-release \
+      && mkdir -p /image-data && chmod 0777 /image-data
```

Comment:
this PR was not meant to solve all problems, just give people starting point.

you can update clam signatures everytime when the image-inspector image starts (as part of the entrypoint). 

i think the disconnect between this and what this is trying to prove is that you are building something that will run as service on every node (via daemon-set) where this is trying to do the "job-based" approach, where you fire up a scan on demand (basically create a "job" with target image to scan and URL where it should post the result).

I dunno our infra details, but I would question why the clamd has to run on every node vs. having specialized nodes intended to do only scanning.